### PR TITLE
fix: ACNA-1810 - update Template Registry API usage for OR category clauses

### PR DESCRIPTION
**Pull request by shazron**
## How Has This Been Tested?

- npm test
- manual test
 
## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [X] All new and existing tests passed.

### CODE REVIEW

File: src/commands/app/add/action.js [hunk 1]
```diff
@@ -94,7 +94,7 @@ class AddActionCommand extends TemplatesCommand {
 
     switch (selection) {
       case 'orgActionTemplates': {
-        const supportedServiceCodes = new Set(orgSupportedServices.map(s => s.code))
+        const supportedServiceCodes = new Set(orgSupportedServices.map(s => `|${s.code}`)) // | symbol denotes an OR clause
```
Comment (moritzraho) [inline src/commands/app/add/action.js:97]: now I am curious 😄  what is this doing ?

Comment (shazron) [reply src/commands/app/add/action.js:97]: it's part of the Template Registry API, to designate that the search query item is an OR clause.
`apis=|foo,|bar` means give me templates with apis either (foo OR bar). If the `|` is omitted, it is by default an AND.
`apis=foo,bar` menas give me templates with apis (foo AND bar).

In this case we want all templates that support any of the services.

**Review by MichaelGoberling — APPROVED** [review]

**Comment by purplecabbage** [comment]
Lgtm!

File: src/commands/app/add/action.js (hunk 1 above)
Comment (moritzraho) [reply src/commands/app/add/action.js:97]: Thanks :)

**Pull request merged**